controllers/routing/routing_controller_test.go

```diff
@@ -97,18 +105,7 @@ var _ = Describe("Platform routing setup for the component", test.EnvTest(), fun
 			})
 
 			// then
-			Eventually(routeExistsFor(svc)).
-				WithContext(ctx).
-				WithTimeout(test.DefaultTimeout).
-				WithPolling(test.DefaultPolling).
-				Should(Succeed())
-
-			Eventually(ingressVirtualServiceExistsFor(svc)).
-				WithContext(ctx).
-				WithTimeout(test.DefaultTimeout).
-				WithPolling(test.DefaultPolling).
-				Should(Succeed())
-
+			externalResourcesShouldExist(ctx, domain, svc)
```

Comment:
If you make `domain` a global var you don't need to pass it around.

Reply:
[d1909f9](https://github.com/maistra/odh-platform/pull/39/commits/d1909f93404529fc63c06db80cb53cee639fae93)